docs/content/preview/yugabyte-platform/security/authentication/oidc-authentication-aad.md

```diff
@@ -154,21 +167,15 @@ To configure YugabyteDB Anywhere for OIDC, you need to be signed in as a Super A
 
```

Comment:
Would put this "To configure YugabyteDB Anywhere for OIDC, you need to be signed in as a Super Admin. You need your Azure application client ID, client secret, and tenant ID." under Configure YugabyteDB Anywhere

Reply:
Would still move this paragraph up to Configure YugabyteDB Anywhere cause I think you need superadmin to set the global flag as well